src/Plugins/Cache.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Peck\Plugins;
+
+final readonly class Cache
+{
+    public function __construct(
+        private string $cacheDirectory,
+    ) {
+        if (! is_dir($this->cacheDirectory)) {
+            mkdir($this->cacheDirectory, 0777, true);
+        }
+    }
+
+    /**
+     * Creates the default instance of Spellchecker.
+     */
+    public static function default(): self
+    {
+        return new self(__DIR__.'/../../.peck.cache');
```

Comment:
I'm unsure about this, but I just have doubts if this works as expected when using the package installed in another project through `./vendor/bin/peck`. Where would this file end up in that case?

Maybe it's better to add something similar to the infer method in the default command to ensure the cache directory lands in the project's root directory?

https://github.com/peckphp/peck/blob/b9e1c0177a1d6619b540433890d1cb965c55b4c7/src/Console/Commands/DefaultCommand.php#L72